app/src/main/java/com/github/se/wanderpals/model/repository/TripsRepository.kt

```diff
@@ -27,7 +27,7 @@ import kotlinx.coroutines.withContext
  *   in short)
  */
 class TripsRepository(
-    private val uid: String,
+    val uid: String,
```

Comment:
Could be changed to private with a getter function to avoid mistakes, works for now

Reply:
Thank you for the suggestion!